src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java

```diff
@@ -96,6 +97,57 @@ public void testAll() throws Exception {
         testDeleteFeedbackSessionsForCourse();
     }
     
+    public void testUpdateSessionResponseRateForDeletedStudentResponse() throws InvalidParametersException,
+                                                                                EntityDoesNotExistException {
+        updateResponseRate_noRemainingResponsesFromStudent_decreaseResponseRate();
+        updateResponseRate_hasRemainingResponsesFromStudent_noChangeToResponseRate();
+    }
+
+    private void updateResponseRate_noRemainingResponsesFromStudent_decreaseResponseRate()
+            throws InvalidParametersException, EntityDoesNotExistException {
+        // response2ForQ2S2C1 is the ONLY response by student1InCourse1 in session2InCourse1
```

Comment:
Let's do this as an assert instead of a comment. So that in future if the test data is modified, it is clearer that student1InCourse1  should only have one response